dbschema/partnership.esdl

```diff
@@ -0,0 +1,34 @@
+module default {
+  type Partnership extending Project::Child {
+    agreementStatus: Partnership::AgreementStatus;
+    mou: File; 
+    mouStatus: Partnership::AgreementStatus;
+    mouStart := .mouStartOverride ?? .project.mouStart;
+    mouEnd := .mouEndOverride ?? .project.mouEnd;
+    mouStartOverride: cal::local_date;
+    mouEndOverride: cal::local_date;
+    agreement: File;
+    required partner: Partner;
+    required organization: Organization;
+    multi types: Partner::Type;
+    financialReportingType: Partnership::FinancialReportingType;
+    primary: bool {
```

Comment:
```suggestion
    required primary: bool {
```
We also do have logic that we should encode here to move this default around, which is currently in app code. That could just be a TODO comment for now.

Reply:
Is this the logic you are referring to: `const primary = isFirstPartnership ? true : input.primary;`?  I'm not sure how to write the expression on the primary bool property to check if this is the first partnership.